validation/README.md

```diff
@@ -2,3 +2,30 @@
 Tests for reflectivity calculations
 (from Andrew Nelson: https://github.com/andyfaff/orso)
 
+## Polarised tests
+### Coordinate system: Refl1D
+The coordinate system used by Refl1D is based on the book chapter "Polarized Neutron Reflectometry" [^1] by C. F. Majkrzak et al. and is defined as seen in this figure: ![NIST_coords](./NIST_coords.png)
+
+The Sample coordinate system is defined such that the surface normal (parallel to Q) is _z<sub>Sample</sub>_, and then _x<sub>Sample</sub>_  and _y<sub>Sample</sub>_ are in the sample plane.
+
+The Lab coordinate system is defined so that _z<sub>Lab</sub>_ points along the polarisation (guide field = _H_) direction.
+
+Further, _x<sub>Lab</sub>_ is defined to be the same as _x<sub>Sample</sub>_.  Thus _x<sub>Sample</sub>_ is a unit vector in the sample plane (perpendicular to Q) that is also perpendicular to the guide field _H_. _y<sub>Sample</sub>_, _y<sub>Lab</sub>_ are then derived from the right-hand-rule from the _x,z_ components in their respective reference frames, and importantly _H_ is then always in the sample _y,z_ plane.
+
+The angle between _z<sub>Sample</sub>_ and _z<sub>Lab</sub>_ we call _AGUIDE_, and the angle between _x<sub>Sample</sub>_ and the in-plane component of the sample magnetization (_M<sub>IP</sub>_) is _theta<sub>M</sub>_.
+
+Because of the way the reference frames are defined, there are always (at least) two valid choices for the direction of _x<sub>Sample</sub>_.
+For some common geometries, here are some corresponding angles:
+#### Guide field H in sample plane (perpendicular to Q)
+For e.g. a vertical-axis right-handed reflectometer, when the guide field at sample points up
+
+ - _AGUIDE_ = pi/2
+ - _theta<sub>M</sub>_ = pi/2 for _M<sub>IP</sub>_ || _H_
```

Comment:
shouldn't theta be zero here? From the diagram theta_m should be pi/2 when it's parallel to y_sam.

Reply:
When H is in the sample plane, it is always parallel (or antiparallel) to y_sam.  So theta_m is pi/2 (or 3pi/2) when M is parallel to H (and y_sam). 

I realize this coordinate system seems somehow inelegant - if we were to redo the whole thing today I would argue that we should choose to define theta_m so that it is zero when aligned with H  when they are both in-plane.  But the coordinate system has been in place for a long long time...

Reply:
- I think I was getting confused because the diagram looks like H is in the XZ plane.
- that alternate definition of angle seems to make more sense to me. However, what you've got at the moment is more analogous to the spherical coordinate system, except with theta and phi swapped around. So if anything I would keep what you have at the moment, but rename theta_m phi_m, and rename phi to theta.

Reply:
Assuming that most analysis packages are setup for AGUIDE=pi/2, is theta_m used in most packages, or 90-theta_m?

Reply:
Looking at the drawing again, I can see what you're saying.  I was trying to make it obvious that H is in the y-z plane but clearly failed.  I'm not quite sure how to make the drawing less ambiguous.

Reply:
See [9f9101c](https://github.com/reflectivity/analysis/pull/66/commits/9f9101c753d8b4b2e9ff4386e6a9b9873d82d147)